apps/cookbook/src/app/showcase/segmented-control-showcase/segmented-control-showcase.component.html

```diff
@@ -165,6 +165,21 @@ <h3>Keyboard support</h3>
     <kbd>End</kbd> moves focus to the last segment button
   </li>
 </ul>
+<h3>
+  Provide custom
+  <code>aria-label</code>
+  for segment buttons
+</h3>
+<p>
+  The optional
+  <code>ariaLabel</code>
+  property of a
+  <code>SegmentItem</code>
+  can be included to provide an
+  <code>aria-label</code>
+  for each segment button. This is useful when the text of the segment button is not descriptive
+  enough on its own.
```

Comment:
Consider a teeny tiny more context to guide the usage a bit.
```suggestion
  for one or more segment buttons. This is useful when the text of the segment button is not 
  descriptive enough on its own when used with Assistive Technology like a screen reader.
```